src/genreclassutil.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import unicode_literals
```

Comment:
Also as pointer: https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement